hbw/app/javascript/packs/hbw/components/form/file_upload.js.jsx

```diff
@@ -256,11 +296,24 @@ modulejs.define('HBWFormFileUpload', ['React'], (React) => {
       return fileReader.readAsBinaryString(file);
     };
 
+    getFileListName = () => {
+      if (typeof this.props.params.file_list_name === 'undefined') {
+        return 'homsOrderDataFileList';
+      } else {
+        return this.props.params.file_list_name;
+      }
```

Comment:
use `in`

on the other hand, I think it's OK here to return `this.props.params.file_list_name || 'homsOrderDataFileList'`

Reply:
I'll use `in` operator